diff -Nru pilot-link.0.9.3/Makefile.in pilot-link.0.9.3.new/Makefile.in
--- pilot-link.0.9.3/Makefile.in	Sat Sep 25 02:20:54 1999
+++ pilot-link.0.9.3.new/Makefile.in	Sat Sep 25 02:24:51 1999
@@ -110,27 +110,32 @@
 	$(SUBMAKE_COMM_CC) libpicc.a
 
 installdirs:
-	$(SHELL) ${srcdir}/scripts/mkinstalldirs $(bindir) $(includedir) $(libdir) $(pixdir) $(mandir1) $(mandir7)
+	$(SHELL) ${srcdir}/scripts/mkinstalldirs $(DESTDIR)$(bindir) \
+		$(DESTDIR)$(includedir) \
+		$(DESTDIR)$(libdir) \
+		$(DESTDIR)$(pixdir) \
+		$(DESTDIR)$(mandir1) \
+		$(DESTDIR)$(mandir7)
 
 install: $(EXECS) installdirs $(PILIB) $(PICCLIB)
-	$(INSTALL_DATA) $(PILIB) $(libdir)/libpisock.la
+	$(INSTALL_DATA) $(PILIB) $(DESTDIR)$(libdir)/libpisock.la
 	if test -n "$(PICCLIB)"; then \
-		$(INSTALL_DATA) $(PICCLIB) $(libdir)/libpicc.la; \
+		$(INSTALL_DATA) $(PICCLIB) $(DESTDIR)$(libdir)/libpicc.la; \
 	fi
 	for p in $(EXECS); do \
-		$(INSTALL_PROGRAM) $$p $(bindir)/$$p; \
+		$(INSTALL_PROGRAM) $$p $(DESTDIR)$(bindir)/$$p; \
 	done
 	for h in include/*.h $(srcdir)/include/*.h include/*.hxx $(srcdir)/include/*.hxx; do \
-		$(INSTALL_DATA) $$h $(includedir)/`basename $$h`; \
+		$(INSTALL_DATA) $$h $(DESTDIR)$(includedir)/`basename $$h`; \
 	done
 	for m in man/*.1; do \
-		$(INSTALL_DATA) $$m $(mandir1)/`basename $$m`; \
+		$(INSTALL_DATA) $$m $(DESTDIR)$(mandir1)/`basename $$m`; \
 	done
 	for m in man/*.7; do \
-		$(INSTALL_DATA) $$m $(mandir7)/`basename $$m`; \
+		$(INSTALL_DATA) $$m $(DESTDIR)$(mandir7)/`basename $$m`; \
 	done
 	for p in $(srcdir)/pix/*.gif $(srcdir)/pix/*.xbm; do \
-		$(INSTALL_DATA) $$p $(pixdir)/`basename $$p`; \
+		$(INSTALL_DATA) $$p $(DESTDIR)$(pixdir)/`basename $$p`; \
 	done
 	$(LIBTOOL) -n --finish $(libdir)
 
