--- pilot-link/pilot-debug.c.orig	Mon Nov 20 12:03:19 2000
+++ pilot-link/pilot-debug.c	Mon Nov 20 12:03:32 2000
@@ -45,9 +45,9 @@
 
 #ifdef TK
 int usetk;
-# include <tk/tk.h>
+# include <tk.h>
 #else
-# include <tcl/tcl.h>
+# include <tcl.h>
 #endif
 
 #ifndef TCL_ACTIVE
--- pilot-link/pd-tty.c.orig	Mon Nov 20 12:03:48 2000
+++ pilot-link/pd-tty.c	Mon Nov 20 12:04:01 2000
@@ -39,9 +39,9 @@
 #ifdef TK
 extern int usetk;
 
-# include <tk/tk.h>
+# include <tk.h>
 #else
-# include <tcl/tcl.h>
+# include <tcl.h>
 #endif
 
 #ifndef TCL_ACTIVE
--- pilot-link/include/pi-header.h.orig	Mon Nov 20 12:36:22 2000
+++ pilot-link/include/pi-header.h	Mon Nov 20 12:35:07 2000
@@ -16,15 +16,9 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
+#ifndef _pi_header_
+#define _pi_header_
+
+void PalmHeader(char *progname);
 
-void PalmHeader(char *progname)
-{
-   fprintf(stderr,"   ---o-o---  The Palm name and logo are copyright Palm Computing, Inc.\n   ---o--o--  and subsidiaries. All rights reserved.\n   ---o-o---\n");
-#ifdef PILOT_LINK_PATCH
-   fprintf(stderr,"   ---o-----  This is %s from pilot-link version %d.%d.%d.%d\n   ---o-----\n", progname, PILOT_LINK_VERSION, PILOT_LINK_MAJOR, PILOT_LINK_MINOR, PILOT_LINK_PATCH);
-   fprintf(stderr,"   ---o-----  pilot-link %d.%d.%d is licensed under the GPL\n               See the file COPYING for more details.\n\n", PILOT_LINK_VERSION, PILOT_LINK_MAJOR, PILOT_LINK_MINOR, PILOT_LINK_PATCH);
-#else
-   fprintf(stderr,"   ---o-----  This is %s from pilot-link version %d.%d.%d\n   ---o-----\n", progname, PILOT_LINK_VERSION, PILOT_LINK_MAJOR, PILOT_LINK_MINOR);
-   fprintf(stderr,"   ---o-----  pilot-link %d.%d.%d is covered under the GPL\n              See the file COPYING for more details.\n\n", PILOT_LINK_VERSION, PILOT_LINK_MAJOR, PILOT_LINK_MINOR);
 #endif
-}
